agents/backport_agent.py

```diff
@@ -288,6 +289,12 @@ def get_prompt() -> str:
       Everything from the previous attempt has been reset. Start over, follow the instructions from the start
       and don't forget to fix the issue.
       {{/build_error}}
+      {{#reproducer_failed}}
+      The reproducer failed after the first backport attempt.
+      Keep it in mind and try to fix the patches you apply to make the reproducer pass.
+      You can inspect the reproducer test at {{tmt_reproducer.git_url}}/blob/{{tmt_reproducer.git_ref}}/{{tmt_reproducer.test}}.
+      The test is located in the {{tmt_reproducer.git_ref}} branch of the {{tmt_reproducer.git_url}} repository.
```

Comment:
Isn't this line redundant? All the info is on the line above. Though I think we should come up with a convention how to reference test cases, for example, inspired by pip and similar (I think we even use something like that in packit):
`https://gitlab.com/redhat/rhel/tests/expat.git@master#testcase=Security/RHEL-114639-CVE-2025-59375-expat-libexpat-in-Expat-allows`